packages/jellyfish-api-core/__tests__/category/mining/getMiningInfo.test.ts

```diff
@@ -39,6 +39,6 @@ describe('Mining', () => {
     expect(info.networkhashps).toBeGreaterThan(0)
     expect(info.pooledtx).toStrictEqual(0)
     expect(info.chain).toStrictEqual('regtest')
-    expect(info.warnings).toStrictEqual('')
+    expect(['', 'This is a pre-release test build - use at your own risk - do not use for mining or merchant applications']).toContain(info.warnings)
```

Comment:
We should not change the order of assertions just so that the logic flows that way since assertions are strictly `expect(something).toBe(expected)`.

For other projects we use extended, https://jest-extended.jestcommunity.dev/docs/matchers/toBeOneOf

Since we don't have it here, I suggest we use Regex Matcher?